bindata/v4.0.0/apiservice-cabundle-controller/clusterrole.yaml

```diff
@@ -13,3 +13,19 @@ rules:
   - watch
   - update
   - patch
+- apiGroups:
+  - ""
+  resources:
+  - configmaps
+  verbs:
+  - get
+  - list
+  - watch
+  - update
+  - create
```

Comment:
Don't we just need access to one specific configmap? Why does it have all of `get, list, watch, update, create`? Wouldn't get/list just be enough?

Reply:
Yes, it makes sense to me. Initially, I thought of giving all the permissions. Let me correct that.